### PR TITLE
Feature/create rest api

### PR DESCRIPTION
**Pull request by Went-Yang**

### CODE REVIEW

File: app/knowledge_service.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+#   id: string,
+#   title: string,
+#   author: string,
+#   created: string,
+#   body: string,
+#   related: string[] //related articles IDs
+# }
+def get_article_by_id(id):
+    d = {}
+    a = Article.query.filter_by(sys_id=id).first()
```
Comment (boyski33) [inline app/knowledge_service.py:14]: Could you rename these to something more descriptive?

File: app/knowledge_service.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+#   title: string,
+#   author: string,
+#   created: string // "2020-08-01"
+# }
+def get_articles_by_date(limit, start):
+    articleObjs =  Article.query.filter(Article.published.isnot(None)).order_by(Article.published.desc()).offset(start).limit(limit)
+    articles = []
+
+    for a in articleObjs:
+        d = {}
```
Comment (boyski33) [inline app/knowledge_service.py:39]: Could you rename these to something more descriptive?

File: app/knowledge_service.py (hunk 1 above)
Comment (Went-Yang) [reply app/knowledge_service.py]: done

File: app/knowledge_service.py (hunk 2 above)
Comment (Went-Yang) [reply app/knowledge_service.py]: done

**Pull request merged**